src/species_initialize.jl

```diff
@@ -173,9 +173,9 @@ function Species(name::String, charge::Int=0, iso::Int=-1)
                 spin = 0.5 * iso
             end
             if anti_atom == false
-                return Species(AS, charge, mass_in_eV, spin, 0, iso) # return the object to track
+                return Species(AS, charge*u"q", mass_in_eV*u"eV/c^2", spin*u"ħ", 0*u"eV/T", iso) # return the object to track
             elseif anti_atom == true
-                return Species("anti-" * AS, charge, mass_in_eV, spin, 0, iso)
+                return Species("anti-" * AS, charge*u"q", mass_in_eV*u"eV/c^2", spin*u"ħ", 0u"eV/T", iso)
```

Comment:
```suggestion
                return Species("anti-" * AS, charge, mass_in_eV, spin, 0, iso)
```

Reply:
Unfortunately that doesn't work in testing. Once I changed the types of the struct, I need to give them units.

Reply:
interesting, i'll take a look later

Reply:
ok yes i get error about dimensional compatbility